erpnext/agriculture/doctype/harvest/harvest.py

```diff
@@ -3,8 +3,62 @@
 # For license information, please see license.txt
 
 from __future__ import unicode_literals
-# import frappe
+import frappe
+import json
 from frappe.model.document import Document
 
 class Harvest(Document):
 	pass
+
+@frappe.whitelist()
+def create_stock_entry(harvest):
+	harvest = frappe.get_doc(json.loads(harvest))
```

Comment:
If you're just doing a `get_doc`, then only pass the harvest name from the frontend. You wouldn't need the `json.loads` then either.

Reply:
we passed json string from frontend that need to convert into dict

Reply:
I'm saying don't pass a dict / object from the frontend, only pass the name. That way, you can direct parse the incoming string without needing to convert it first.